paket.dependencies

```diff
@@ -1,14 +0,0 @@
-source https://nuget.org/api/v2
-
-nuget FAKE
-nuget FSharp.Core
-nuget FluentAssertions
-nuget xunit
-nuget xunit.runner.visualstudio version_in_path: true
-nuget xunit.runner.console
-nuget FSharp.Formatting
-nuget FsCheck
-
-github gsscoder/CSharpx src/CSharpx/Maybe.cs
-github gsscoder/CSharpx src/CSharpx/EnumerableExtensions.cs
-github gsscoder/railwaysharp src/RailwaySharp/ErrorHandling.cs

```

Comment:
I'm not sure we can get rid of paket so easily, these files get pulled in an compiled into the project.

And this is why paket works so much better than Nuget, because Nuget constantly made assumptions that were just not good in the real world.

Reply:
right, I added these few files manually and removed Paket as with PackageReference and the new csproj sdk format it's unnecessary.

Reply:
What do you think about copying those three files into our source tree permanently? All were written by @gsscoder (the original author of commandlineparser) and I don't expect any new updates to the source. The only question is the legality of copying those files, as neither of the external projects are tagged with a license.

Reply:
there is no reason to use Paket here as the source code files probably won't change anymore (they didn't for multiple years). Also csharpx clearly states that the files can either be consumed via Paket or by manually copying them:

> Source files are conceived to be manually added to other projects or by setting a reference via Paket. See these paket.dependencies and paket.references as example.

https://github.com/gsscoder/CSharpx#csharpx

I suggest to not make things more complicate as they need to be and just use nuget and get rid of Paket.